paddle/optimizer/lr_policy.h

```diff
@@ -0,0 +1,45 @@
+#ifndef PADDLE_OPTIMIZER_LR_POLICY_H_
+#define PADDLE_OPTIMIZER_LR_POLICY_H_
+
+#include <algorithm>
+#include "OptimizerConfig.pb.h"
+
+namespace paddle {
+namespace optimizer {
+
+class BaseLr {
+public:
+  BaseLr(double lr) : learning_rate(lr) {}
+  virtual ~BaseLr() {}
+  virtual double get_learning_rate(const uint64_t num_sample_passed) = 0;
```

Comment:
I thought it is a base class, derived classes named with the convention of XXXLr.   fix done.